streaming/java/streaming-api/src/main/java/org/ray/streaming/jobgraph/VertexType.java

```diff
@@ -6,6 +6,6 @@
 public enum VertexType {
   MASTER,
   SOURCE,
-  PROCESS,
+  TRANSFORMATION,
```

Comment:
```suggestion
  TRANSFORM
```

Reply:
Why use verb but not noun？

Reply:
Please see google dataflow paper

Reply:
Transform could be used as noun, e.g.  Fourier transform.